### PR TITLE
Add memory ballast extension skeleton

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: extension/ballastextension/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Health Check
```
Comment (tigrannajaryan) [inline extension/ballastextension/README.md:1]: Wrong name.

**Review by tigrannajaryan — COMMENTED** [review]
What's the plan with existing command line option? Do we keep it? If both are set do we have 2 ballasts?

File: extension/ballastextension/README.md (hunk 1 above)
Comment (bogdandrutu) [reply extension/ballastextension/README.md]: Done.

**Comment by bogdandrutu** [comment]
@tigrannajaryan 
> What's the plan with existing command line option? Do we keep it? If both are set do we have 2 ballasts?
Let's discuss this in #1960

**Comment by bogdandrutu** [comment]
@tigrannajaryan ping

**Pull request closed without merging**